src/fits.cc

```diff
@@ -1072,7 +1072,7 @@ void writeKeyFromProperty(Fits &fits, daf::base::PropertySet const &metadata, st
         }
     } else {
         // FIXME: inherited this error handling from fitsIo.cc; need a better option.
-        LOGLS_WARN("afw.writeKeyFromProperty",
+        LOGLS_WARN("lsst.afw.writeKeyFromProperty",
```

Comment:
This logger name is different from all others, I presume this is intentional?

Also maybe define a constant for "lsst.afw.fits" logger name instead of repeating it 10 times?

Reply:
I wasn't sure how much I wanted to change the logger names but I'll take a second look at this one. It probably should have `fits` in it.